app/waf/src/models/adc.model.ts

```diff
@@ -148,11 +176,86 @@ export class Adc extends CommonEntity {
   })
   status: ConfigTypes['status'];
 
-  // @property({
-  //   type: 'object',
-  //   required: true,
-  // })
-  // onBoarding: ConfigBody['onBoarding'];
+  @property({
+    type: 'object',
+    schema: {
+      create: true,
+      response: true,
+      example: {
+        declaration: {
+          system: {
+            license: {
+              regKey: 'Q2431-06294-92275-11650-8028186',
```

Comment:
It is OK to let user to retrieve onboarding data. That would be a useful way for them to quickly know the configuration of their device. However, we may need to consider to hide some information. For example, I am not quite if we can let a tenant user know the license key. Maybe not.

Reply:
Yeah, we can support retrieving some of the information. However, after all we need not user to provide the detailed information.